pkg/resource/deploy/plan_executor.go

```diff
@@ -174,7 +176,7 @@ func (pe *planExecutor) Execute(callerCtx context.Context, opts Options, preview
 						pe.reportError(pe.plan.generateEventURN(event.Event), resErr)
 					}
 					cancel()
-					return false, result.TODO()
+					return false, result.FromError(result.TODO())
```

Comment:
not sure hte best thing to do here.  Now that we can actually return a result, i'm happy to do so.  i'm just not sure what error should go in it.

Reply:
@swgillespie for this bit.

Reply:
this beomces a Bail